### DISCUSSION TITLE
Feature Request: Redirects

### DISCUSSION THREAD

**Discussion by jonathan-bird** [aerni/statamic-advanced-seo#104] [Ideas]
A big thing I face with my clients is setting up page redirects - both the initial set of redirects manually set to migrate their current Wordpress or other platform sites to their new site, and ongoing in terms of them moving pages around and expecting automated 301 redirects.

Would be amazing if this Advanced SEO plugin would consider adding Redirects as a feature, similar to https://statamic.com/addons/candour/aardvark-seo

**Comment by aerni** [comment]
I don't think I'll ever add that feature. It's simply out of scope of what I want this addon to be. And it would add a whole other layer of complexity. I always use [Redirect](https://statamic.com/addons/rias/redirect).

**Reply by jonathan-bird** [reply]
Fair enough, I thought it would compliment well as you've already gone down the Sitemap route, and saves yet another addon. 

But all good :)

**Reply by aerni** [reply]
Yeah, I hear you. I've been back and forth about this decision. And sitemaps have a whole lot more to do with SEO in my opinion. So it was a no-brainer to add that feature.

**Reply by jonathan-bird** [reply]
Yeah I can understand that. I honestly think it'd give you a big leg up from the other SEO addons (namely SEO Pro) if it had it too. 

But again, totally understand your perspective and it's your addon, you can take it any direction you like. 

You've definitely got my support for all Statamic sites moving forward.

**Comment by aerni** [comment]
I'm currently working on a redirects feature. Will be the next big feature coming to this addon.
